.github/workflows/pull_request.yml

```diff
@@ -107,19 +111,31 @@ jobs:
       - name: install obspy
         shell: bash -l {0}
         run: |
-          pip install -e .
+          export PATH="$pythonLocation/bin/:$PATH"
+          python -m pip install -e .
 
       - name: install obspy github api
         shell: bash -l {0}
-        run : |
-          pip install git+https://github.com/obspy/obspy_github_api@${OBSPY_GITHUB_BRANCH}
+        run: |
+          export PATH="$pythonLocation/bin/:$PATH"
+          python -m pip install git+https://github.com/obspy/obspy_github_api@${OBSPY_GITHUB_BRANCH}
 
       - name: run test suite
         shell: bash -l {0}
         run: |
+          export PATH="$pythonLocation/bin/:$PATH"
           export MODULELIST=`obshub read-config-value module_list --path=${CONFIG_PATH}`
           export MODULELISTSPACES=`obshub read-config-value module_list_spaces --path=${CONFIG_PATH}`
           coverage run --rcfile=.coveragerc --source=${MODULELIST} -m obspy.scripts.runtests --no-flake8 -n gh-actions -r --ci-url="${CI_URL}" --pr-url="${PR_URL}" $MODULELISTSPACES
+      
+      - name: parse coverage results
```

Comment:
Duplicate job?

Reply:
Nope, previously it was a just as a command following the testrunner command. I had issues that some failing tests were not shown as failed jobs so I extracted this parsing of coverage to a separate step